indico/web/client/js/react/components/PersonDetailsModal.jsx

```diff
@@ -128,51 +130,52 @@ export default function PersonDetailsModal({
         </Translate>
       )}
       <Form.Group widths="equal">
-        <Form.Field>
-          <Translate as="label" context="Salutation">
-            Title
-          </Translate>
-          <FinalDropdown
-            name="title"
-            fluid
-            search
-            selection
-            options={titles}
-            placeholder={Translate.string('None', 'Title (salutation)')}
-          />
-        </Form.Field>
+        <FinalDropdown
+          name="title"
+          label={Translate.string('Title', 'Salutation')}
+          fluid
+          search
+          selection
+          options={titles}
+          placeholder={Translate.string('None', 'Title (salutation)')}
+          required={requiredPersonFields?.includes('title')}
+        />
         {!extraParams?.disableAffiliations && (
-          <Form.Field>
-            <Translate as="label">Affiliation</Translate>
-            <FinalAffiliationField hasPredefinedAffiliations={hasPredefinedAffiliations} />
-          </Form.Field>
+          <FinalAffiliationField
+            label={Translate.string('Affiliation')}
+            hasPredefinedAffiliations={hasPredefinedAffiliations}
+            required={requiredPersonFields?.includes('title')}
```

Comment:
Should this be `affiliation`?
```suggestion
            required={requiredPersonFields?.includes('affiliation')}
```

Reply:
oops, yep